frontend/src/components/Election/Voting/GenericBallotView.js

```diff
@@ -144,26 +142,56 @@ const Rows = ({ candidates, scores, onClick, columns }) =>
         onClick={(score) => onClick(n, score)}
         columns={columns}
       />
-      <Divider style={{ width: '100%' }} />
+      <Divider className="rowDivider"/>
     </>
   ));
 
 // Represents the list of column headings for all possible scores
-const ColumnHeadings = ({columns}) => (
+const ColumnHeadings = ({starHeadings, columns, leftTitle, rightTitle, headingPrefix}) => (
   <>
-    <Grid container alignItems="stretch">
+  { leftTitle != '' &&
+    <Grid container alignItems="stretch" >
       <Grid item xs={5}></Grid>
-      <ScoreColumnHeadings columns={columns}/>
+      <Typography align='center' className="columnDescriptor">
```

Comment:
Should put the leftTitle and rightTitle typeographies in a <Grid item xs={1}>, should help align with the other rows below.